resume.md

```diff
@@ -1,49 +1,52 @@
-# FIRST LAST NAME
+# Adithya Kumar
 
-(123) 456 - 7890 | email@gmail.com
+(512) 660 - 1911 | adithya.shak.kumar@gmail.com
 
-www.website.com | github.com/myusername | linkedin.com/myusername
+github.com/shakstzy | https://www.linkedin.com/in/adithyashakthikumar/
 
 ## EDUCATION
-**University of College** | B.S. Degree ... *MONTH 20XX*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
+University of California, Berkeley | B.S. Electrical Engineering & Computer Science + Business ... *May 2023*
+* **Coursework**: Data Structures, Algorithms,Structure and Interpretation of Computer Programs, Multivariable Calculus, Probability and Statistics
 
 ## SKILLS
-* **Languages**: Python, Java, C, SQL, ...
-* **Libraries/Frameworks**: Pandas, TensorFlow, PyTorch, ...
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
-* *...*
+* **Languages**: Python, Java, C++, SQL, Javascript, HTML, CSS
+* **Libraries/Frameworks**: React, TensorFlow, PyTorch
+* **Version Control**: Git, GithubDesktop
 
 ## EXPERIENCE
-**Software Engineer**: Company | City, State ... *Month 20XX - Present*
-* Implemented XYZ Microservice using TECHNOLOGY to delivery PRODUCT to 123 Customers Globally
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
+**Project Management Intern**: Intensivate | Berkeley, CA ... *Sep. 2020 – Present*
+* Led international team of 6 to develop an internal framework host server to power real-time collaboration software using JS & Nginx
+* Facilitated biweekly standup meetings, established communication between team & client contact, and defined/tracked deliverables
```

Comment:
Each and every point should further the case why you are the best technical candidate for the role.  Don't be afraid to put very technical name drops into your bullet points.

What is the impact?  can this be measured?